Sources/Sentry/SentryBinaryImageCache.m

```diff
@@ -35,8 +36,21 @@ - (void)stop
 
 - (void)binaryImageAdded:(const SentryCrashBinaryImage *)image
 {
+    if (image->name == NULL) {
```

Comment:
Is it possible for `image` to also be `NULL`? If `nextNode->image` for whatever reason in NULL, then we could pass a NULL pointer and dereferencing here would crash. https://github.com/getsentry/sentry-cocoa/blob/24c239b8e33dc88bfc70f588069a029eb00b4165/Sources/SentryCrash/Recording/SentryCrashBinaryImageCache.c#L143-L147

Reply:
I think it shouldn't but a check for NULL does not harm. Thanks for pointing that out.

Reply:
I think it shouldn't but a check for NULL does not harm. Thanks for pointing that out.